.changeset/dry-turtles-sit.md

```diff
@@ -0,0 +1,6 @@
+---
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Remove ratable characteristic; add --uitk-measured-foreground-hover, --uitk-measured-foreground-active, --uitk-measured-foreground-undo, --uitk-measured-foreground-activeDisabled, --uitk-palette-measured-foreground-active, --uitk-palette-measured-foreground-activeDisabled
```

Comment:
I find this style of changeset hard to read. What do you think of using something like:
```suggestion
Remove ratable characteristic
Add new tokens to measured characteristic:
- --uitk-measured-foreground-hover
- --uitk-measured-foreground-active
- --uitk-measured-foreground-undo
- --uitk-measured-foreground-activeDisabled
- --uitk-palette-measured-foreground-active
- --uitk-palette-measured-foreground-activeDisabled
```

Reply:
Probably worth listing out the ratable tokens that have been removed

Reply:
haha agreed, will change now, it's because i did it quickly and lazily in the terminal :P